[rum] document missing data if page opened in background (#11438)
DataDog/documentation: content/en/real_user_monitoring/browser/monitoring_page_performance.md

| [First Input Delay][6]        | Interactivity    | Time elapsed between a user’s first interaction with the page and the browser’s response.             | <100ms      |
| [Cumulative Layout Shift][7]  | Visual stability | Quantifies unexpected page movement due to dynamically loaded content (for example, third-party ads) where 0 means no shifts happening. | <0.1        |

**Notes**:

> - First Input Delay and Largest Contentful Paint are not collected for pages opened in background (i.e. new tab or window without focus).

Reviewer: ```suggestion
- First Input Delay and Largest Contentful Paint are not collected for pages opened in the background (for example, in a new tab or a window without focus).
```
